### ISSUE TITLE
land options seperate from AGCM.rc desired

### ISSUE THREAD

**Issue by JustinPerket** [GEOS-ESM/UMD_Etc#8]
@smahanam has code in cvs that takes catch and catchCN-related options out of AGM.rc and into a dedicated land resource file. Previous objections about duplication of LDAS.rc are now moot since LDAS.rc is now in the GEOSldas repo. It would be trivial to implement.

**Comment by mathomp4** [comment]
@JustinPerket Did you mean to do this in UMD? I'm surprised the ocean das cares about land parameters

<sub>Sent with <a href="http://githawk.com">GitHawk</a></sub>

**Comment by smahanam** [comment]
These are optional physics parameters used by land models : Catchment, CatchmentCN, Route or any land physics parameterization. 
That is, they are used by programs inside GEOSland_GridComp and up.

**Comment by smahanam** [comment]
Matt, this is how we have implemented this in AGCM:
gcm_run.j concatenates RC/GEOS_LandGridComp.rc with AGCM.rc inside the "scratch" directory. 

GEOSldas sets ROOT_CF to LDAS.rc so that is independent of this change.

That said, all other systems like, ADAS etc, we will need to make similar changes to their gcm_run.j equivalent script to concatenate RC/GEOS_LandGridComp.rc and AGCM.rc (or the main resource file)

**Comment by mathomp4** [comment]
Ah. Okay. I had no idea UMD had an AGCM.rc. Learning every day. 😃 

<sub>Sent with <a href="http://githawk.com">GitHawk</a></sub>

**Comment by JustinPerket** [comment]
@mathomp4, you are right. I was looking for what repo AGCM.rc went into, and found UMD first. I really wanted GEOSgcm_App.
